census/src/test/scala/model/RssdToMsaSpec.scala

```diff
@@ -0,0 +1,13 @@
+package model
+
+import hmda.census.model.RssdToMsa
+import org.scalatest.{ MustMatchers, WordSpec }
+
+class RssdToMsaSpec extends WordSpec with MustMatchers {
+  "RssdToMsa" must {
+    "map values correctly" in {
+      val map = RssdToMsa.map
+      map.keys.toList.length mustBe 123703
```

Comment:
The `branches_msa.csv` file is 201221 lines long, but it produces a map that only has 123703 keys. Is that a concern? Are we losing information from the file when we make this map?